### PR TITLE
core dayController code

### PR DESCRIPTION
**Pull request by breim9**
(Practice making a PR) 
Get the current date, make date formats as used by the code in other places, respond accordingly if is a new day

### CODE REVIEW

**Comment by shoelessness** [comment]
Feel free to request me as a reviewer, so you can see what the intended flow of a PR is 💯

File: stacks-v3/src/App.js
Comment (shoelessness) [inline stacks-v3/src/App.js]: Perhaps this could be a template string?

```suggestion
    let currentDate = `${thisDay}/${thisMonth}/${thisYear}`;
```

File: stacks-v3/src/App.js
Comment (shoelessness) [inline stacks-v3/src/App.js]: This could be encapsulated to its own function:

```
if (this.isNewDay(currentDate, visDate)) {
  this.onNewDay()
}
```

As you may want this exact sequence to run from somewhere else in the application. Your thoughts?

**Review by shoelessness — CHANGES_REQUESTED** [review]
I think this is looking pretty good! Just a few tweaks and you're good to go :shipit:

File: stacks-v3/src/App.js
Comment (breim9) [reply stacks-v3/src/App.js]: Makes sense!

**Comment by breim9** [comment]
So I made changes based on your review and committed them - in a proper PR would I then wait for you to close the review or should I go ahead and dismiss it myself? 
ie. it still shows one change requested after I've addressed the change. 
image

**Review by shoelessness — APPROVED** [review]

**Comment by shoelessness** [comment]
That's about right! Normally, in a group project, the ability to merge the PR would be blocked until another team member had reviewed and approved it.

**Pull request merged**
